docs/source/tutorials/structure_your_input.rst

```diff
@@ -114,58 +81,162 @@ each image a so-called "weak-label" indicating to which subdirectory it belongs.
+   from lightly.transforms import SimCLRTransform
+
+   transform = SimCLRTransform(input_size=32, gaussian_blur=0.0)
+   dataset = torchvision.datasets.CIFAR10(
+       "datasets/cifar10", download=True, transform=transform
+   )
+
+
+Hugging Face Datasets
+--------------------
+
+To use a dataset from the Hugging Face Hub 🤗, we can simply apply the desired transformations using the
+`set_transform <https://huggingface.co/docs/datasets/v2.20.0/en/package_reference/main_classes#datasets.Dataset.set_transform>`_
+helper method and then create a native PyTorch dataloader.
+
+
+.. code-block:: python
+
+    import torch
+    from typing import Any, Dict
+    from datasets import load_dataset
+    from lightly.transforms import SimCLRTransform
+
+    dataset = load_dataset("uoft-cs/cifar10", trust_remote_code=True)
+
+    ## Use pre-defined set of transformations from Lightly SSL
+    transform = SimCLRTransform()
+
+    def apply_transform(batch: Dict[str, Any])-> Dict[str, Any]:
+        """
+        Applies the given transform on all elements of batch["image"].
+        """
+        assert "image" in example_batch, "batch must contain key 'image'"
+        batch["image"] = [transform(img.convert("RGB")) for img in batch["image"]]
+        return batch
+
+    dataset.set_transform(apply_transform)
+    dataloader = torch.utils.data.DataLoader(dataset["train"])
+
+Image Augmentations
+-------------------
+
+Many SSL methods leverage image augmentations to better learn invariances in the training process. For example,
+by using different crops of a given image, the SSL model will be trained to produce a representation
+that is invariant to these different crops. When using a operation such as grayscale or colorjitter as augmentation,
+it will produce a representation that is invariant to the color information [1]_.
+
+We can use off the shelf augmentations from libraries like `torchvision transforms <https://pytorch.org/vision/stable/transforms.html>`_
+and `albumentations <https://albumentations.ai/docs/>`_ or the ones offered by Lightly SSL's
+`transforms <https://docs.lightly.ai/self-supervised-learning/lightly.transforms.html>`_ submodule while creating our datasets.
+
+.. code-block:: python
+
+    import albumentations as A
+    import torchvision.transforms as T
+    from albumentations.pytorch import ToTensorV2
+
+    from lightly.data import LightlyDataset
+
+    ## Torchvision Transforms
+    torchvision_transform = T.Compose(
+        [
+            T.RandomHorizontalFlip(),
+            T.ToTensor(),
+        ]
+    )
+
+    ## Albumentation Transforms
+    albumentation_transform = A.Compose(
+        [
+            A.CenterCrop(height=128, width=128),
+            A.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)),
+            ToTensorV2(),
+        ]
+    )
+
+    ## Lightly Transforms
+    lightly_transform = SimCLRTransform()
+
+    dataset = LightlyDataset(input_dir="image_dir/", transform=torchvision_transform)
```

Comment:
Suggestion to make it more evident that datasets/transforms are interchangeable:

```suggestion
    
    # Datasets and transforms can be mixed and matched together:
    dataset = LightlyDataset(input_dir="image_dir/", transform=torchvision_transform)
    dataset = torchvision.datasets.CIFAR10("datasets/cifar10", transform=lightly_transform)
```